src/DataFactory/DataFactoryV2/help/Get-AzDataFactoryV2DataFlow.md

```diff
@@ -0,0 +1,185 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.DataFactoryV2.dll-Help.xml
+Module Name: Az.DataFactory
+online version: https://docs.microsoft.com/en-us/powershell/module/az.datafactory/get-azdatafactoryv2dataflow
+schema: 2.0.0
+---
+
+# Get-AzDataFactoryV2DataFlow
+
+## SYNOPSIS
+Gets information about data flows in Data Factory.
+
+## SYNTAX
+
+### ByFactoryName (Default)
+```
+Get-AzDataFactoryV2DataFlow [[-Name] <String>] [-ResourceGroupName] <String> [-DataFactoryName] <String>
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### ByFactoryObject
+```
+Get-AzDataFactoryV2DataFlow [[-Name] <String>] [-DataFactory] <PSDataFactory>
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### ByResourceId
+```
+Get-AzDataFactoryV2DataFlow [-ResourceId] <String> [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+The Get-AzDataFactoryV2DataFlow cmdlet gets information about data flows in Azure Data Factory.
+If you specify the name of a data flow, this cmdlet gets information about that data flow.
+If you do not specify a name, this cmdlet gets information about all the data flows in the data factory.
+
+## EXAMPLES
+### Example 1: Get information about all data flows
+```powershell
+PS C:\> Get-AzDataFactoryV2DataFlow -ResourceGroupName "ADF" -DataFactoryName "WikiADF"
+
+DataFlowName      : TaxiDemo1
+ResourceGroupName : adf
+DataFactoryName   : WikiADF
+Properties        : Microsoft.Azure.Management.DataFactory.Models.MappingDataFlow
+
+DataFlowName      : dataflow1
+ResourceGroupName : adf
+DataFactoryName   : WikiADF
+Properties        : Microsoft.Azure.Management.DataFactory.Models.MappingDataFlow
+
+DataFlowName      : dataflow3
+ResourceGroupName : adf
+DataFactoryName   : WikiADF
+Properties        : Microsoft.Azure.Management.DataFactory.Models.MappingDataFlow
+```
+
+This command gets information about all data flows in the data factory named WikiADF.
+
+### Example 2: Get information about a specific data flow
+```powershell
+PS C:\> Get-AzDataFactoryV2DataFlow -ResourceGroupName "ADF" -DataFactoryName "WikiADF" -Name "dataflow1"
+
+DataFlowName      : dataflow1
```

Comment:
list format.  You will need to have a format entry for this in order to have a table format, see: https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/generate-format.ps1xml-file.md

Reply:
Oh I see. I thought I need to include a example to show how to convert to a table format result 😅, will update my pr soon

Reply:
The generation tool doesn't work with "Could not load file or assembly 'System.Runtime.Loader, Version=4.0.0.0, Culture=neutral,
PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified." error (I tried to change the version to 4.0.0 in csproj and I got a different error with something about assembly context). For now I just added manually into format file instead and it should still work if someone uses the script to generate the file in the future.